packages/compass-preferences-model/src/preferences-schema.ts

```diff
@@ -311,6 +312,17 @@ export const storedUserPreferencesProps: Required<{
     validator: z.string().uuid().optional(),
     type: 'string',
   },
+  /**
+   * Stores the timestamp for when the user was created
+   */
+  userCreatedAt: {
```

Comment:
This is a new prop in preferences, but this changes is 100% bacwards compatible because we always sync it from the existing user storage entry on the application start, preferences here just make it possible to access this outside of main process